s3-commons/build.gradle.kts

```diff
@@ -18,10 +18,13 @@ plugins { id("aiven-apache-kafka-connectors-all.java-conventions") }
 
 val amazonS3Version by extra("1.12.777")
 val amazonSTSVersion by extra("1.12.777")
+val amazonV2Version by extra("2.29.34")
```

Comment:
This can be updated to `2.29.36`  :laughing:   Please try to keep up!!

(Just kidding, but we should keep in mind to bump it to the most recent patch version before release!)